eng/scripts/Compare-CurrentToCodegeneration.ps1

```diff
@@ -59,11 +61,10 @@ class GenerationInformation {
 function Find-GenerationInformation {
   param (
     [System.Collections.ArrayList]$GenerationInformations,
-    [string]$ServiceDirectory,
-    [string]$RegenerationType
```

Comment:
Just curious - how many regeneration types do we currently support? Are we consolidating them under same code logic?

Reply:
Only Swagger, TypeSpec, or All will be supported, and this shouldn't increase any time soon.

We could consolidate further in the future but Swagger and TypeSpec code generation use very different semantics.